doc/man7/EVP_KDF-KB.pod

```diff
@@ -84,9 +97,37 @@ Label "label", and Context "context".
 
  EVP_KDF_CTX_free(kctx);
 
+This example derives 10 bytes using FEEDBACK-CMAC-AES256, with KI "secret",
+Label "label", and IV "sixteen bytes iv".
+
+ EVP_KDF *kdf;
```

Comment:
Most of the examples include the snippet of code to fetch the EVP_KDF and create the EVP_KDF_CTX.  Plus freeing them later.